static/css/timetable/partials/course_modal.scss

```diff
@@ -173,8 +186,7 @@
     font-family: $font-body;
     font-size: 16px;
     font-weight: 400;
-    margin: 10px 0;
-    padding-right: 15%;
+    margin: 0px 0px 3px 0px;
```

Comment:
`margin-bottom: 3px`

Reply:
When I change this to just a margin-bottom property, the modal sub-title becomes drastically misaligned (see below). Think it's worth just leaving it like this. 

<img width="752" alt="Screen Shot 2019-06-12 at 8 44 02 PM" src="https://user-images.githubusercontent.com/43327083/59395610-d7de1000-8d52-11e9-8f1e-6543d527b56f.png">